sass/style.scss

```diff
@@ -34,6 +34,7 @@ Nicolas Gallagher and Jonathan Neal http://necolas.github.com/normalize.css/
 @import "typography/typography";
 
 @import "layout/work-page";
+@import "layout/contactUs_aboutUs-page";
```

Comment:
Hey Brendan,

I've try avoid caps in filenames. Windows doesn't recognise the difference but Git and Macs do. 
It's good to use camelCase for variables / functions in javascript but filenames and CSS classes should be lower case and hyphen (-) / underscore separated.